modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractDartCodegen.java

```diff
@@ -112,13 +111,13 @@ public AbstractDartCodegen() {
         }
         setReservedWordsLowerCase(reservedWordsList);
 
+        // These types return isPrimitive=true in templates
         languageSpecificPrimitives = Sets.newHashSet(
                 "String",
                 "bool",
                 "int",
                 "num",
-                "double",
-                "dynamic"
```

Comment:
What happend with `dynamic`?

Reply:
Well, dynamic is not actually used as a type anywhere, so it doesn't need to be handled at all.

Reply:
What if a property is named "dynamic"? This wouldn't escaped anymore and would result in e.g. `String dynamic` which wouldn't compile

Reply:
In contrast to other types, `dynamic` is actually a keyword and part of the reserved word list. So this can not happen. This list here only, additionally defines language primitives, which `dynamic` is not.

Reply:
Ah, I didn't recognized the separate `dart-keywords.txt` file which includes `dynamic`. Then never mind. I just searched inside the class